cmd/minishift/cmd/stop.go

```diff
@@ -61,12 +61,16 @@ func runStop(cmd *cobra.Command, args []string) {
 
 	fmt.Println("Stopping local OpenShift cluster...")
 
-	if !skipUnRegistration {
-		if _, err := cluster.UnRegister(api); err != nil {
+	if !skipUnRegistration && minishiftConfig.InstanceConfig.IsRegistered {
+		supportUnregistration, err := cluster.UnRegister(api)
```

Comment:
If `minishiftConfig.InstanceConfig.IsRegistered` was the whole point, it seems we neglected to actually check the state of this variable. This is why this change is suggested.

Without checking this, it means that images like B2D and CentOS will always be requested about `supportRegistration`. If for some reason, deployment did not succeed in a working VM (no IP for instance), it becomes problematic for the user to stop (or delete) the VM. This should never happen...